model/src/test/kotlin/config/OrtConfigurationTest.kt

```diff
@@ -146,9 +146,7 @@ class OrtConfigurationTest : WordSpec({
                     useSsl shouldBe true
                     fromAddress shouldBe "no-reply@oss-review-toolkit.org"
                 }
-            }
```

Comment:
Nit: This commit is unrelated to the topics of the "surrounding" commits. While it's perfectly fine to still keep it as part of this PR, maybe just make it the last commit, so it does not "disrupt" the review flow for the "real" topic of this PR.